app/src/main/res/layout/activity_display_favorites.xml

```diff
@@ -22,11 +22,39 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent">
 
-        <android.support.v7.widget.RecyclerView
-            android:id="@+id/favorites_recycler_view"
+        <LinearLayout
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:scrollbars="vertical" />
+            android:orientation="vertical">
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:id="@+id/show_all_favorites_layout"
+                android:background="@color/background_blue"
```

Comment:
If you use a color variable, don't put "blue" as part of the name. The idea of use a color var is to allow future changes, only on the value.